### PR TITLE
[es] Replace mdn.mozillademos.org images that exist in content

### PR DESCRIPTION
**Pull request by caugner**
### Description

Replaces occurrences of images that reference https://mdn.mozillademos.org/files/ with the (normalized) filename, but only those that exist in mdn/content (at the same slug location).

(Missing images are added in https://github.com/mdn/translated-content/pull/10480.)

### Motivation

The domain no longer serves images (it used to redirect).

### Additional details

**Note**: Unfortunately, the "Preview URLs" don't include images from content, so the only way to test that this works is to check out this PR and visit the URLs locally.

### Related issues and pull requests

Extracted from https://github.com/mdn/translated-content/pull/10402.
Part of https://github.com/mdn/translated-content/issues/10400.

### CODE REVIEW

**Comment by caugner** [comment]
The failing Markdownlint check is unrelated. 🤷

File: files/es/mozilla/add-ons/webextensions/what_are_webextensions/index.md [hunk 1]
```diff
@@ -10,37 +10,37 @@ Una extensión agrega características y funciones a un explorador. Se ha creado
...
 
-![](https://mdn.mozillademos.org/files/15807/ublock_origin_add_on.png)
+![](ublock_origin_add_on.png)
 
 **Ejemplos:** [Origen uBlock](https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/), [Lector](https://addons.mozilla.org/en-US/firefox/addon/reader/) y [Caja de Herramientas para Google Play Store™](https://addons.mozilla.org/en-US/firefox/addon/toolbox-google-play-store/)
 
 **Agregar herramientas y nuevas funciones de navegación**: añada nuevas funciones a un taskboard o genere imágenes de código QR a partir de URLs, hipervínculos o texto de página. Con opciones de interfaz de usuario flexibles y la potencia de las [APIs de WebExtensions](/es/docs/Mozilla/Add-ons/WebExtensions), puede añadir fácilmente nuevas funcionalidades a un navegador. Usted puede realzar casi cualquier característica o funcionalidad de sitios web, no tiene que ser su web.
 
-**![](https://mdn.mozillademos.org/files/15806/QR_Code_Image_Generator_add_on.png)**
+**![](qr_code_image_generator_add_on.png)**
```
Comment (Graywolf9) [inline files/es/mozilla/add-ons/webextensions/what_are_webextensions/index.md:31]:
```suggestion
![](qr_code_image_generator_add_on.png)
```

File: files/es/web/progressive_web_apps/installing/index.md [hunk 2]
```diff
@@ -28,50 +28,50 @@ _La interfaz de usuario para esto varía de un navegador a otro, pero la idea ge
 
 _En un dispositivo Android que usa Firefox, verás un ícono de "inicio" con un ícono más (+) en su interior; este es el ícono "Agregar a la pantalla de inicio" que se muestra para cualquier sitio que tenga las características necesarias en su lugar._
 
-_**![Una captura de pantalla de un navegador web que muestra el ícono Agregar a la pantalla de inicio en la parte superior](<https://mdn.mozillademos.org/files/17158/android-a2hs-icon.png >)**_
+_**![Una captura de pantalla de un navegador web que muestra el ícono Agregar a la pantalla de inicio en la parte superior](<android-a2hs-icon.png >)**_
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/installing/index.md:31]:
```suggestion
![Una captura de pantalla de un navegador web que muestra el ícono Agregar a la pantalla de inicio en la parte superior](android-a2hs-icon.png)
```

File: files/es/web/progressive_web_apps/installing/index.md [hunk 3]
```diff
@@ -28,50 +28,50 @@ _La interfaz de usuario para esto varía de un navegador a otro, pero la idea ge
...
 
 _Al tocar este, se mostrará un pasquín de confirmación; al presionar el botón grande "+ AGREGAR A LA PANTALLA DE INICIO", se completa la acción y se agrega la aplicación a la pantalla de inicio. Ten en cuenta que en Android 8 y versiones posteriores, se mostrará primero un cuadro de diálogo de permiso "Agregar a la pantalla de inicio" a nivel del sistema._
 
-_![Una captura de pantalla de Firefox para Android que solicita confirmación antes de instalar una aplicación web](https://mdn.mozillademos.org/files/17160/fx-a2hs-banner.png)_
+_![Una captura de pantalla de Firefox para Android que solicita confirmación antes de instalar una aplicación web](fx-a2hs-banner.png)_
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/installing/index.md:35]:
```suggestion
![Una captura de pantalla de Firefox para Android que solicita confirmación antes de instalar una aplicación web](fx-a2hs-banner.png)
```

File: files/es/web/progressive_web_apps/installing/index.md [hunk 4]
```diff
@@ -28,50 +28,50 @@ _La interfaz de usuario para esto varía de un navegador a otro, pero la idea ge
...
 
 ### Navegador de Internet Samsung
 
 _Si tiene Samsung Internet en tu dispositivo móvil, verás un icono diferente._
 
-_![Una captura de pantalla del navegador de Internet de Samsung que muestra su icono Agregar a la pantalla de inicio](https://mdn.mozillademos.org/files/17161/samsung-internet-add-app.png)_
+_![Una captura de pantalla del navegador de Internet de Samsung que muestra su icono Agregar a la pantalla de inicio](samsung-internet-add-app.png)_
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/installing/index.md:41]:
```suggestion
![Una captura de pantalla del navegador de Internet de Samsung que muestra su icono Agregar a la pantalla de inicio](samsung-internet-add-app.png)
```

File: files/es/web/progressive_web_apps/installing/index.md [hunk 5]
```diff
@@ -28,50 +28,50 @@ _La interfaz de usuario para esto varía de un navegador a otro, pero la idea ge
...
+_En iOS de Apple (incluidos iPhoneOS y iPadOS), el navegador Safari integrado en el dispositivo tiene cierto soporte para aplicaciones web, incluido el soporte para agregar a la función de pantalla de inicio. Para agregar una aplicación web a la pantalla de inicio (también conocida como lanzador o trampolín), toca el botón para compartir (![Icono de compartir cuadrado y flecha desde iOS y macOS](square.svg)) en la parte inferior de la pantalla:_
 
-_![Captura de pantalla que muestra la ubicación del botón Compartir en Safari en iOS 13](https://mdn.mozillademos.org/files/17163/safari-ios-a2hs-icon.png)_
+_![Captura de pantalla que muestra la ubicación del botón Compartir en Safari en iOS 13](safari-ios-a2hs-icon.png)_
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/installing/index.md:59]:
```suggestion
![Captura de pantalla que muestra la ubicación del botón Compartir en Safari en iOS 13](safari-ios-a2hs-icon.png)
```

File: files/es/web/progressive_web_apps/installing/index.md [hunk 6]
```diff
@@ -28,50 +28,50 @@ _La interfaz de usuario para esto varía de un navegador a otro, pero la idea ge
...
 
 _Esto llama al panel para compartir. Entre las opciones debe estar la opción "Agregar a la pantalla de inicio", a menos que el usuario la haya eliminado específicamente de la lista al editar las opciones mostradas:_
 
-_![Captura de pantalla del panel para compartir de Safari iOS, que muestra la opción](https://mdn.mozillademos.org/files/17165/safari-ios-share-menu.png)_
+_![Captura de pantalla del panel para compartir de Safari iOS, que muestra la opción](safari-ios-share-menu.png)_
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/installing/index.md:63]:
```suggestion
![Captura de pantalla del panel para compartir de Safari iOS, que muestra la opción](safari-ios-share-menu.png)
```

File: files/es/web/progressive_web_apps/installing/index.md [hunk 7]
```diff
@@ -28,50 +28,50 @@ _La interfaz de usuario para esto varía de un navegador a otro, pero la idea ge
...
 
 _Al elegir "Agregar a la pantalla de inicio" aquí se presenta el cuadro de diálogo de confirmación, que no solo confirma que el usuario desea agregar la aplicación a la pantalla de inicio, sino que también le permite personalizar su nombre._
 
-_![Una captura de pantalla de iOS Safari que muestra su panel de configuración y confirmación "Agregar a la pantalla de inicio"](https://mdn.mozillademos.org/files/17167/safari-ios-a2hs-banner.png)_
+_![Una captura de pantalla de iOS Safari que muestra su panel de configuración y confirmación "Agregar a la pantalla de inicio"](safari-ios-a2hs-banner.png)_
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/installing/index.md:67]:
```suggestion
![Una captura de pantalla de iOS Safari que muestra su panel de configuración y confirmación "Agregar a la pantalla de inicio"](safari-ios-a2hs-banner.png)
```

File: files/es/web/progressive_web_apps/installing/index.md [hunk 8]
```diff
@@ -28,50 +28,50 @@ _La interfaz de usuario para esto varía de un navegador a otro, pero la idea ge
...
 
 _Al hacer clic en "Agregar", la aplicación se incluye en la pantalla de inicio._
 
 ### La aplicación web instalada
 
 _Independientemente del navegador y el dispositivo que estés utilizando, cuando eliges agregar la aplicación a tu pantalla de inicio o al iniciador de aplicaciones, aparecerá allí con el mismo aspecto que cualquier aplicación. Normalmente, esto significa que verás el icono de la aplicación junto con un título corto._
 
-_![Captura de pantalla de una pantalla de inicio de Android con la aplicación "Foxes" mostrada](https://mdn.mozillademos.org/files/17157/a2hs-on-home-screen.png)_
+_![Captura de pantalla de una pantalla de inicio de Android con la aplicación "Foxes" mostrada](a2hs-on-home-screen.png)_
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/installing/index.md:75]:
```suggestion
![Captura de pantalla de una pantalla de inicio de Android con la aplicación "Foxes" mostrada](a2hs-on-home-screen.png)
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much! Everything looks good, I used this PR to fix all the images involved in the PR 

Also add `favicon32.png` image that was missing in `content` too: https://github.com/mdn/content/pull/22752 for /es/docs/web/css/background-repeat

**Pull request merged**